src/core/virtual_sites/lb_inertialess_tracers.cpp

```diff
@@ -45,6 +45,21 @@ void GetIBMInterpolatedVelocity(const Utils::Vector3d &p, double *v,
 
 bool *isHaloCache = nullptr;
 
+namespace {
+bool in_local_domain(Utils::Vector3d const &pos) {
```

Comment:
here it is... if this would be in the lattice class... or we check the node-locality independent on the lattice?

Reply:
In it's current form it depends on lb globals, so there is some work to be done here. That was clearly out of scope.

Reply:
So abstractly what is missing is a function that checks if a point is in a rectangle (and some more box functions, like volume)

Reply:
[just geometry stuff](https://www.boost.org/doc/libs/1_70_0/libs/geometry/doc/html/geometry/reference/algorithms.html)